public/docs/_examples/toh-5/ts/app/app.component.1.ts

```diff
@@ -1,14 +1,42 @@
 // #docplaster
-// #docregion
+// #docregion , v2
 import { Component } from '@angular/core';
 
+// #enddocregion v2
 @Component({
   selector: 'my-app',
   template: `
     <h1>{{title}}</h1>
     <my-heroes></my-heroes>
   `
 })
+// #enddocregion
+// #docregion v2
+@Component({
```

Comment:
Does having the duplicate `Component` decorators cause problems with the live code?